[2024-08-28T08:27:35.983+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-28T08:27:36.020+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: country_data_pipeline.load_data manual__2024-08-28T08:16:43.543250+00:00 [queued]>
[2024-08-28T08:27:36.051+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: country_data_pipeline.load_data manual__2024-08-28T08:16:43.543250+00:00 [queued]>
[2024-08-28T08:27:36.052+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-28T08:27:36.122+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): load_data> on 2024-08-28 08:16:43.543250+00:00
[2024-08-28T08:27:36.130+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=471) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-28T08:27:36.130+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'country_data_pipeline', 'load_data', 'manual__2024-08-28T08:16:43.543250+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dags.py', '--cfg-path', '/tmp/tmpmsdn_6lc']
[2024-08-28T08:27:36.132+0000] {standard_task_runner.py:72} INFO - Started process 480 to run task
[2024-08-28T08:27:36.132+0000] {standard_task_runner.py:105} INFO - Job 124: Subtask load_data
[2024-08-28T08:27:36.230+0000] {task_command.py:467} INFO - Running <TaskInstance: country_data_pipeline.load_data manual__2024-08-28T08:16:43.543250+00:00 [running]> on host c63ae693b587
[2024-08-28T08:27:36.396+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='country_data_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T08:16:43.543250+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-28T08:16:43.543250+00:00'
[2024-08-28T08:27:36.398+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-28T08:27:37.153+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-28T08:27:37.167+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-28T08:27:37.168+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=country_data_pipeline, task_id=load_data, run_id=manual__2024-08-28T08:16:43.543250+00:00, execution_date=20240828T081643, start_date=20240828T082736, end_date=20240828T082737
[2024-08-28T08:27:37.234+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-28T08:27:37.262+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-28T08:27:37.265+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
